### DISCUSSION TITLE
[Q&A] NetDiscovery from Linux agent

### DISCUSSION THREAD

**Discussion by JustApu** [glpi-project/glpi-agent#573] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I have installed the agent via glpi-agent-1.7.1-linux-installer.pl but "glpi-agent --list-tasks" only reports RemoteInventory (v1.4) and Inventory (v1.15) as available tasks. Do I need the task modules (specifically NetDiscovery) installed manually? I was under the impression they should be installed automatically when using that script.

``RPM packages are arch independents and installation may require some repository setups, better use [glpi-agent-1.7.1-linux-installer.pl](https://github.com/glpi-project/glpi-agent/releases/download/1.7.1/glpi-agent-1.7.1-linux-installer.pl) when possible.``

**Comment by g-bougard** [comment, marked as answer]
Hi @JustApu 

run `perl glpi-agent-1.7-linux-installer.pl --help` as explained in doc: https://glpi-agent.readthedocs.io/en/latest/installation/index.html#linux-perl-installer

You will see all possible options and there one option to define what to install: `--type=network` in your case

**Reply by JustApu** [reply]
Thank you, @g-bougard. That was what I missed.

**Comment by sta-zot** [comment]
Hi. I have installed from the DEB package whether it is possible to include the necessary modules in the configuration.
